components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/utils/APIUtil.java

```diff
@@ -10825,29 +10825,49 @@ public static KeyManagerConnectorConfiguration getKeyManagerConnectorConfigurati
 
     /**
      * Extract Endpoint Registry Entry information
-     * @param endpointConfig
-     * @return Map
+     *
```

Comment:
change naming in description

Reply:
Fixed in https://github.com/wso2/carbon-apimgt/pull/8728/commits/216e08f31573e70a482b6f9e3235b139d06c07bf